lms/djangoapps/experiments/views.py

```diff
@@ -16,11 +21,17 @@ class ExperimentDataViewSet(viewsets.ModelViewSet):
     permission_classes = (permissions.IsAuthenticated, IsStaffOrOwner,)
     queryset = ExperimentData.objects.all()
     serializer_class = ExperimentDataSerializer
+    _cached_users = {}
```

Comment:
Is this object re-created with every request? Or should our cache be scoped to the method call?